recipe/meta.yaml

```diff
@@ -128,7 +129,6 @@ outputs:
     test:
       requires:
         - {{ compiler('cxx') }}
-        - sysroot_linux-64 2.17  # [linux64 and cdt_name!='cos6']
```

Comment:
I think I had to add stdlib c for the tests too since they test for the ability to compile